src/PowerShell/PowerShellExtensions.cs

```diff
@@ -37,5 +37,13 @@ public static Collection<T> InvokeAndClearCommands<T>(this PowerShell pwsh)
                 pwsh.Commands.Clear();
             }
         }
+
+        public static string FormatObjectToString(this PowerShell pwsh, object inputObject)
+        {
+            return pwsh.AddCommand("Write-Output")
```

Comment:
Can you please add a comment about why we want to call "write-output"?

Reply:
```suggestion
            // PowerShell's `Out-String -InputObject` handles collections differently than when receiving InputObjects from the pipeline. (i.e. `$collection | Out-String`).
            // That is why we need `Write-Output` here.
            // See related GitHub issue here:
            // https://github.com/PowerShell/PowerShell/issues/8246
            return pwsh.AddCommand("Write-Output")
```